package.json

```diff
@@ -19,13 +19,13 @@
     "grpc-web": "1.0.7"
   },
   "scripts": {
-    "build": "tsc -d",
-    "clean": "rm -rf ./generated ./dist ./build",
-    "pretest": "./scripts/regenerate_protos.sh && tsc",
+    "build": "npm run clean && ./scripts/regenerate_protos.sh && npm run lint && tsc -d",
+    "webpack": "npm run clean && ./scripts/regenerate_protos.sh && npm run lint && npx webpack",
```

Comment:
I get that now we have `ts-loader` we don't _need_ to run `tsc`, but wouldn't it be nice to make sure our TypeScript compiles without errors?

Maybe we can have a command that calls both `build` and `webpack`? (Might need to rename some of the commands).

What do you think? Or am I being overly cautious?

Reply:
I think it is overly cautious as each CI has to pass all compilations.

I agree that this would be nice to have nice to have and doesn't increase our build times significantly so I added in a tsc --noEmit to the webpack script and the test script.